modules/codechanges.adoc

```diff
@@ -55,7 +55,7 @@ Username: {{GITLAB_USER}}
 Password: {{GITLAB_PASSWORD}}
 ====
 
-Click the Settings link on the top right of the screen, and then click "Webhooks":
+Click the Settings link on the top right of the screen, and then click "Integrations":

```

Comment:
Is this a change in Gitlab? When I did this the other day it still said "Webhooks".